VocaDbWeb/Controllers/Api/AntiforgeryApiController.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.AspNetCore.Antiforgery;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+using VocaDb.Web.Code.Security;
+using ApiController = Microsoft.AspNetCore.Mvc.ControllerBase;
+
+namespace VocaDb.Web.Controllers.Api;
+
+[EnableCors(AuthenticationConstants.WebApiCorsPolicy)]
+[Route("api/antiforgery")]
+[ApiController]
+public class AntiforgeryApiController : ApiController
+{
+	private readonly IAntiforgery _antiforgery;
+
+	public AntiforgeryApiController(IAntiforgery antiforgery)
+	{
+		_antiforgery = antiforgery;
+	}
+
+	[HttpGet("token")]
+	[Authorize]
+	[EnableCors(AuthenticationConstants.AuthenticatedCorsApiPolicy)]

```

Comment:
The API server must have CORS set to allow only trusted sites. This is to prevent third-party AJAX requests from getting hold of a CSRF token. See [CSRF tokens for SPAs - Michael Zanggl](https://michaelzanggl.com/articles/csrf-tokens-for-spas/).